### PR TITLE
Add info for CVE-2021-25748.

### PR DESCRIPTION
**Pull request by cjcullen**

### CODE REVIEW

**Comment by cjcullen** [comment]
Warning is expected

File: 2021/25xxx/CVE-2021-25748.json [hunk 1]
```diff
@@ -1,18 +1,101 @@
...
+                        "value": "CWE-20: Improper Input Validation"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "refsource": "MISC",
```
Comment (tallclair) [inline 2021/25xxx/CVE-2021-25748.json:84]: Not sure it matters, but we usually use `MLIST` for the mailing list announcement, and `CONFIRM` for the issue

Comment (SaranBalaji90) [reply 2021/25xxx/CVE-2021-25748.json:84]: @cjcullen do you want to change this to keep it consistent - https://github.com/kubernetes/committee-security-response/blob/main/cna-handbook.md#populate-cve-details

**Comment by SaranBalaji90** [comment]
Running retest just to get test output files and update comments with actual failure message for future reference. Most probably this will fail due to product name mismatch which is expected in this case.

**Comment by SaranBalaji90** [comment]
```
 Checking 2021/25xxx/CVE-2021-25748.json ...
 WARNING: expected .affects.vendor.vendor_data[0].product.product_data[0].product_name to be 'Kubernetes'; found 'Kubernetes ingress-nginx'
FAIL
```

**Pull request merged**